src/app/app.module.ts

```diff
@@ -103,6 +104,7 @@ const enableDevTools = environment && environment.enableDevTools;
     { provide: HAMMER_GESTURE_CONFIG, useClass: IonicGestureConfig },
     OutcomeBehaviourMapProvider,
     LogHelper,
+    RekeySearchProvider,
```

Comment:
Does it need to be a global provider can it not be in the RekeySearch Page Module

Reply:
Moved the declaration to the RekeySearchPage module.